core/resources/shaders/point.fs

```diff
@@ -48,8 +48,16 @@ void main(void) {
             float c2 = circle(uv, vec2(0.0), circleRadiusOut);
             color = vec4(vec3(c1) * v_color.rgb, c2 * v_alpha * v_color.a);
         #else
+            // premultiplied texture color
             vec4 texColor = texture2D(u_tex, v_texcoords);
-            color = vec4(texColor.rgb * v_color.rgb, v_alpha * texColor.a * v_color.a);
+            //color = vec4(texColor.rgb * v_color.rgb, texColor.a * v_color.a);
+
+            // Correct? v_color as multipler - to say how much of each texture channel is used?
+            color = texColor * v_color;
+            color *= v_alpha;
```

Comment:
@hjanetzek I just tested this it seems correct

```
draw: 
    icons:
        sprite: museum
        color: [1.0, 1.0, 1.0, 0.5]
```

Current:
![screen shot 2016-03-03 at 10 11 46 am](https://cloud.githubusercontent.com/assets/7061573/13498742/3475a226-e129-11e5-9e10-8f53da14899c.png)

Proposed:
![screen shot 2016-03-03 at 10 12 32 am](https://cloud.githubusercontent.com/assets/7061573/13498749/3ab05622-e129-11e5-892c-fe08b0289727.png)